docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+| **Ethereum Provider** | A JS object (`window.ethereum`) injected into each web page opened in the browser providing web3 compatible provider. |
+| **Bridge** | A set of facilities allow bidirectional communication between JS code and Status. |
+
+## Implementation
+Status uses a [forked version](https://github.com/status-im/react-native-webview) of [react-native-webview](https://github.com/react-native-community/react-native-webview)  to display web or dapps content. The fork provides an Android implementation of JS injection before page load. It is required in order to properly inject Ethereum Provider object.
+
+Status injects two JS scripts: 
+  - [provider.js](https://github.com/status-im/status-react/blob/develop/resources/js/provider.js): `window.ethereum` object
+  - [webview.js](https://github.com/status-im/status-react/blob/develop/resources/js/webview.js): override for `history.pushState` used internally
+
+Dapps running inside a browser communicate with Status Ethereum node by means of a *bridge* provided by react-native-webview library. The bridge allows for bidirection communication between browser and Status. In order to do so, it injects a special `ReactNativeWebview` object into each page it loads. 
+
+On Status (React Native) end, `react-native-webview` library provides `WebView.injectJavascript` function on a webview component that allows to execute arbitrary code inside the webview. Thus it is possible to inject a function call passing Status node response back to the Dapp.
+
+Below is the table briefly describing what functions/properties are used. More details available in package [docs](https://github.com/react-native-community/react-native-webview/blob/master/docs/Guide.md#communicating-between-js-and-native).
+
+| Direction | Side | Method |
+|-----------|------|-----------
+| Browser->Status | JS | `ReactNativeWebView.postMessage()`
+| Browser->Status | RN | `WebView.onMessage()`
+| Status->Browser | JS | `ReactNativeWebView.onMessage()`
+| Status->Browser | RN | `WebView.injectJavascript()`
+
+
+
+## Usage in Dapps
+
+`window.ethereum`
+
+### Properties
+
+#### `isStatus`
+Returns true.
+
+#### `status`
+Returns a `StatusAPI` object. For now it supports one method: `getContactCode` that sends a `contact-code` request to Status.
+
+
+
+### Methods
+
+#### `isConnected`
+Similarly to Ethereum JS API [docs](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3isconnected), it should be called to check if connection to a node exists. On Status, this fn always returns true, as once Status is up and running, node is automatically started.
+
+#### `scanQRCode`
+Sends a `qr-code` Status API request.
+
+#### `enable` (DEPRECATED)
+Sends a `web3` Status API request. It returns a first entry in the list of available accounts.
+
+Legacy `enable` method as defined by [EIP1102](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1102.md).
+
+#### `send` (DEPRECATED)
```

Comment:
Shouldn't the `(DEPRECATED)` be removed here?

Reply:
@jakubgs In latest EIP-119e release https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1193.md `send` is marked as DEPRECATED to be replaced by `request`, if i understand correctly.

Reply:
Fair enough.